content/sql/concepts/window-functions/terms/lag/lag.md

```diff
@@ -29,33 +32,106 @@ OVER ( [ partition_by ] order_by )
 - `partition_by` - Allows the result set to be grouped based on a column. This is an optional argument, if not specified the result set will be treated as a single group.
 - `order_by` - Determines the order of the result set. If `partition_by` is specified, it will order the grouped data instead.
 
-## Example
+## Examples
 
-The example below uses the `LAG()` function to create the new column `previous_age` based on values in the previous row of the `Users` table.
+The examples below use the table Sales, which contains orders of different products that are sold on a particular date and their price that they are sold to.
 
-Users Table
+Sales Table
 
-| first_name | last_name | age |
-| ---------- | --------- | --- |
-| chris      | smith     | 30  |
-| dave       | james     | 19  |
-| jenna      | black     | 35  |
-| kyle       | xy        | 60  |
+| product_id | order_date | sold_price |
+| ---------- | ---------- | -------- |
+|          1 | 2024-01-01 |      100 |
+|          1 | 2024-02-01 |      120 |
+|          1 | 2024-03-01 |       80 |
+|          2 | 2024-01-01 |       50 |
+|          2 | 2024-02-01 |       60 |
+|          2 | 2024-02-02 |       60 |
+
+Example 1 : 
+
+The below example uses `LAG()` on column `order_date` with Order By `order_date`.
```

Comment:
We can simplify the flow of the sentence by replacing `Order by` with ordered by, as follows -

```suggestion
The below example uses `LAG()` on column `order_date`, ordered by `order_date`.
```